src/config/app.ts

```diff
@@ -10,6 +10,8 @@ export const defaultConfig = yaml.load(defaultContent)
 
 const MS_IN_HOUR = 60 * 60 * 1000
 
+export const MEMO_SHARING_SATS_THRESHOLD = 1000
```

Comment:
please use config not a const

Reply:
But even when moving it to config lets provide a default because needing to provision the value in config will delay rollout.

Reply:
Moved the value to `default.yml` file https://github.com/GaloyMoney/galoy/pull/415/commits/7b1982ccb8eded01be8da5d50b08e83483dbd265